doc/index.rst

```diff
@@ -3,9 +3,9 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-==========================================================================
-Python implementation of elastic-net regularized generalized linear models
```

Comment:
@pavanramkumar I removed elastic-net from the title. I think we have no expanded beyond just elastic nets. If you're not okay with it, I can put it back.